x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -243,14 +233,13 @@ export class HeadlessChromiumDriver {
     });
   }
 
-  private registerListeners(conditionalHeaders: ConditionalHeaders, logger: Logger) {
+  private registerListeners(url: string, customHeaders: Headers, logger: Logger) {
     if (this.listenersAttached) {
       return;
     }
 
-    // @ts-ignore
     // FIXME: retrieve the client in open() and  pass in the client
-    const client = this.page._client;
+    const client = this.page.client();
```

Comment:
Nice job finding this undocumented method :) 

Looks like it is documented in the types exported by Puppeteer, but is not documented at https://pptr.dev/#?product=Puppeteer&version=v10.2.0&show=api-class-page